Tests/UnitTests/Scripts/tests.js

```diff
@@ -211,6 +211,205 @@ describe("clearTimeout", function () {
     });
 });
 
+// URL
+describe("URL", function () {
+
+    // Currently all of the properties that the polyfill has implemented
+    function checkURL(
+        url,
+        href,
+        hostname,
+        origin,
+        pathname,
+        search) {
+        expect(url).to.have.property("hostname", hostname);
+        expect(url).to.have.property("href", href);
+        expect(url).to.have.property("origin", origin);
+        expect(url).to.have.property("pathname", pathname);
+        expect(url).to.have.property("search", search);
+    }
+
+    // Base URLs:
+    const baseUrl = 'https://developer.mozilla.org';
```

Comment:
Does `httpbin.org` not work for these tests? I would prefer to use this instead of pinging some public server that isn't intended for testing.

Reply:
It works, for now we are making any request to the server, but I`ve changed to httpbin